lm_eval/models/huggingface.py

```diff
@@ -290,8 +299,10 @@ def _create_auto_model(
                 load_in_8bit=load_in_8bit,
                 trust_remote_code=trust_remote_code,
                 torch_dtype=torch_dtype,
+                use_auth_token=token,
                 **model_kwargs,
             )
+            logging.info(model.hf_device_map)
```

Comment:
Does this need more text explaining what is being logged here, or is it pretty obvious?

Reply:
I will add more info, but I will keep the log. it's useful for debugging.